### PR TITLE
Creating Object/Fields for Attendance Events

### LINKED ISSUE #723: Create Attendance Event Object and Fields
**New HEDA Object:** We are creating object and fields to support attendance events.

* **What is the motivation / use case for changing the behavior?**
An *attendance* event records a student’s presence/participation in either an entire school day, or class period. Can represent both absences as well as tardies. *Attendance* can be assumed if no absence is reported. For our MVP we are focusing on storing *attendance* NOT taking *attendance*. 

Note: This work item will focus on schema only. Logic/Automation are slated for farther down the road. 

* **Acceptance Criteria**
TBD

### PR DESCRIPTION
**Pull request by xinnanlisf**
# Critical Changes

# Changes
- We've added a new Attendance Event object to track a student's tardiness or absence for a class period or an entire school day. You can use it to monitor the frequency and reasons for students' Attendance Events, and whether they are excused or unexcused.
We don't recommend using Attendance Events to track students' expected attendance at school.
To display Attendance Event information for a Contact, add the Attendance Event related list to the Contact page layout.

# Issues Closed
Fixes #723
# New Metadata
Object:
Attendance_Event__c.object

Translation files:
Attendance_Event__c-ca
Attendance_Event__c-en_US
Attendance_Event__c-es
Attendance_Event__c-fr

Layout: 
Attendance_Event__c-HEDA Attendance Event Layout

Tab:
Attendance_Event__c.tab
# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: src/objects/Attendance_Event__c.object [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+        <inlineHelpText>The reason for the tardy/absence.</inlineHelpText>
+        <label>Reason</label>
+        <length>32768</length>
+        <trackTrending>false</trackTrending>
+        <type>LongTextArea</type>
+        <visibleLines>3</visibleLines>
+    </fields>
+    <fields>
+        <fullName>Start_Time__c</fullName>
+        <description>The time when the pre-excused absence will start.</description>
```
Comment (lifewithryan) [inline src/objects/Attendance_Event__c.object]: May want to change the description here a bit to match the description for end time "For pre-excused absences..."

File: src/package.xml [hunk 2]
```diff
@@ -729,6 +744,7 @@
         <members>Account-HEDA Organization Layout</members>
         <members>Address__c-HEDA Address Layout</members>
         <members>Affiliation__c-HEDA Affiliation Layout</members>
+        <members>Attendance_Event__c-Attendance Event Layout</members>
```
Comment (robertbwatson) [inline src/package.xml]: @xinnanlisf @lifewithryan This is interesting -- I would say that we should be consistent here with the naming and call it the "HEDA Attendance Event Layout" ... thoughts? We might want to discuss in Quip/standup.

File: src/objects/Attendance_Event__c.object (hunk 1 above)
Comment (xinnanlisf) [reply src/objects/Attendance_Event__c.object]: Should be good now! @lifewithryan

File: src/package.xml (hunk 2 above)
Comment (xinnanlisf) [reply src/package.xml]: Yep, good catch!

**Comment by robertbwatson** [comment]
@xinnanlisf I think we should include the new metadata here (similar to #726) and have Julie write up some release notes for us. Not sure when that should happen - I've been trying to do it before my PRs get merged though.

**Comment by xinnanlisf** [comment]
Morning @lifewithryan, I made a minor change to the page layout. fyi

File: src/package.xml [hunk 3]
```diff
@@ -794,6 +809,7 @@
         <members>Account-HEDA Organization Layout</members>
         <members>Address__c-HEDA Address Layout</members>
         <members>Affiliation__c-HEDA Affiliation Layout</members>
+        <members>Attendance_Event__c-HEDA Attendance Event Layout</members>
```
Comment (robertbwatson) [inline src/package.xml]: I know this has been in progress since pre-EDA, but similar to the other PR this should be EEEEEEDA :)

**Comment by xinnanlisf** [comment]
Should we close this too since we will need to rework on attendance object/fields? @lifewithryan

**Pull request closed without merging**